x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -0,0 +1,531 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { run } from '@kbn/dev-cli-runner';
+import yargs from 'yargs';
+import _ from 'lodash';
+import globby from 'globby';
+import pMap from 'p-map';
+import { ToolingLog } from '@kbn/tooling-log';
+import { withProcRunner } from '@kbn/dev-proc-runner';
+import cypress from 'cypress';
+import { findChangedFiles } from 'find-cypress-specs';
+import grep from '@cypress/grep/src/plugin';
+import crypto from 'crypto';
+import fs from 'fs';
+import { createFailError } from '@kbn/dev-cli-errors';
+import axios from 'axios';
+import path from 'path';
+import os from 'os';
+import { renderSummaryTable } from './print_run';
+import type { SecuritySolutionDescribeBlockFtrConfig } from './utils';
+import { isSkipped, parseTestFileConfig } from './utils';
+
+interface ProductType {
+  product_line: string;
+  product_tier: string;
+}
+
+interface CreateEnvironmentRequestBody {
+  name: string;
+  region_id: string;
+  product_types: ProductType[];
+}
+
+interface Environment {
+  name: string;
+  id: string;
+  region: string;
+  es_url: string;
+  kb_url: string;
+  product: string;
+}
+
+interface Credentials {
+  username: string;
+  password: string;
+}
+
+const DEFAULT_REGION = 'aws-eu-west-1';
+let log: ToolingLog;
+
+/**
+ * Retrieve test files using a glob pattern.
+ * If process.env.RUN_ALL_TESTS is true, returns all matching files, otherwise, return files that should be run by this job based on process.env.BUILDKITE_PARALLEL_JOB_COUNT and process.env.BUILDKITE_PARALLEL_JOB
+ */
+const retrieveIntegrations = (integrationsPaths: string[]) => {
```

Comment:
NIT: This method is used as well in `parallel.ts` let's extract it to the `utils.ts`.

Reply:
retrieveIntegrations was moved to the utils.ts